src/atoms/index.js

```diff
@@ -2,5 +2,5 @@ import { atom } from 'recoil';
 
 export const userStoredList = atom({
   key: 'userStoredList',
-  default: [],
+  default: localStorage.getItem('userList') || [],
```

Comment:
`Recommend-Must`
추후 회의때 나온 트리구조에 맞춰 constants/localStorage.js 에서 값을 읽어오면 좋을 것 같습니다.
```jsx
import { USER_STORED_LIST } from '../constants/localStorage'
```
```suggestion
  default: localStorage.getItem(USER_STORED_LIST) || [],
```

Reply:
네 수정하였습니다!